flow/Makefile

```diff
@@ -102,6 +102,24 @@ endif
 SHELL          ?= /bin/bash
 .SHELLFLAGS    = -o pipefail -c
 
+# Test that the shell works. Otherwise, warn user and fall back to bash. Note
+# that Make does not inherit $SHELL from the parent environment (See 5.3.2 of
+# GNU Make documentation).
+$(shell echo "" &> /dev/null)
+ifneq ($(.SHELLSTATUS),0)
+  define shell_warn_msg
+[WARNING][FLOW] Shell '$(SHELL)' returned an error on a simple test. Attempting\
+to fall back to '/bin/bash'.
+To fix this warning, try:
+  1) Ubuntu/WSL users: run 'sudo dpkg-reconfigure dash' and select 'No'
+  2) Run 'chsh' and change your shell path to '/bin/bash' or another shell
```

Comment:
Just a small improvement: give the full command to `chsh` similar to `dpkg`.

```
chsh -s /bin/bash
```
?